.github/CONTRIBUTING.md

```diff
@@ -5,6 +5,36 @@ appreciated and welcomed. If you're planning a new feature or changing
 the API, please create an issue first. This way we can ensure that your precious
 work is not in vain.
 
+Table of Contents
+
+- [Issues](#issues)
+- [Your first Contribution](#your-first-contribution)
+- [Setup](#setup)
+    - [Using npm (setup)](#using-npm-setup)
+    - [Using yarn (setup)](#using-yarn-setup)
+- [Testing](#testing)
+    - [Using npm (testing)](#using-npm-testing)
+    - [Using yarn (testing)](#using-yarn-testing)
+- [Editor Config](#editor-config)
+- [Dependencies](#dependencies)
+- [Branching Model](#branching-model)
+- [Naming a branch](#naming-a-branch)
+    - [Features](#features)
+    - [Fixes](#fixes)
+- [Testing](#testing-1)
+- [Pull Requests](#pull-requests)
+- [Submitting a good Pull Request](#submitting-a-good-pull-request)
+- [Commit message format](#commit-message-format)
+    - [Commit Message Format](#commit-message-format)
+- [Migrating with the CLI](#migrate-with-the-cli)
```

Comment:
IMO we can keep "Migrating with the CLI" and remove the sub-entries from the TOC.
Also, this link doesn't work